	- pass CC, CFLAGS and LDFLAGS when running subdir configure and make
	- install without root-permissions, do not create device nodes
	- need to pass PPPVERSIONS to subtargets
--- capi4k-utils.orig/Makefile	2002-11-20 14:45:14.000000000 +0100
+++ capi4k-utils/Makefile	2011-07-15 14:15:21.446001582 +0200
@@ -53,7 +53,7 @@ ifneq ($(SUBDIRS),)
 endif
 
 subtargets: $(CONFIGURATION)
-	set -e; for i in `echo $(SUBDIRS)`; do $(MAKE) -C $$i all; done
+	set -e; for i in `echo $(SUBDIRS)`; do CC="${CC}" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" PPPVERSIONS="${PPPVERSIONS}" $(MAKE) -C $$i all; done
 
 rootperm:
 	@echo 'main(int argc,char**argv){unlink(argv[0]);return(getuid()==0);}'>g
@@ -62,16 +62,10 @@ rootperm:
 		exit 1; \
 	fi
 
-install: rootperm
+install:
 	set -e; for i in `echo $(SUBDIRS)`; do $(MAKE) -C $$i install; done
-	@if [ -c $(DESTDIR)/dev/isdnctrl0 ] && ls -l $(DESTDIR)/dev/isdnctrl0 | egrep "[[:space:]]45,[[:space:]]+64[[:space:]]" > /dev/null; \
-	then \
-		echo -e '(some) ISDN devices already exist, not creating them.\nUse scripts/makedev.sh manually if necessary.'; \
-	else \
-		sh scripts/makedev.sh $(DESTDIR) ; \
-	fi
 
-uninstall: rootperm
+uninstall:
 	set -e; for i in `echo $(SUBDIRS)`; do $(MAKE) -C $$i uninstall; done
 
 #
@@ -134,7 +128,7 @@ subconfig: scripts/autoconf.h
 	@set -e; for i in `echo $(SUBDIRS)`; do \
 		if [ -x $$i/configure ] ; then \
 			echo -e "\nRunning configure in $$i ...\n"; sleep 1; \
-			(cd $$i; ./configure --sbindir=$(CONFIG_SBINDIR) --bindir=$(CONFIG_BINDIR) --mandir=$(CONFIG_MANDIR) || $(MAKE) -C ../ ERRDIR=$$i cfgerror); \
+			(cd $$i; CC="${CC}" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" ./configure --sbindir=$(CONFIG_SBINDIR) --bindir=$(CONFIG_BINDIR) --mandir=$(CONFIG_MANDIR) || $(MAKE) -C ../ ERRDIR=$$i cfgerror); \
 		elif [ -f $$i/Makefile.in ] ; then \
 			echo -e "\nRunning make -f Makefile.in config in $$i ...\n"; sleep 1; \
 			$(MAKE) -C $$i -f Makefile.in config; \
